qutip/bloch.py

```diff
@@ -463,7 +478,10 @@ def render(self, fig=None, axes=None):
             self.axes.set_zlim3d(-0.7, 0.7)
         # Manually set aspect ratio to fit a square bounding box.
         # Matplotlib did this stretching for < 3.3.0, but not above.
-        self.axes.set_box_aspect((1, 1, 1))
+        if parse_version(matplotlib.__version__) >= parse_version('3.3'):
+            self.axes.set_box_aspect((1, 1, 1))
+        else:
+            pass
```

Comment:
`else: pass` is unnecessary - it's fine to just have the `if ...:` part of the statement

Reply:
Thanks!